include/grpc/support/time.h

```diff
@@ -47,7 +47,11 @@ extern "C" {
 
 typedef struct gpr_timespec {
     time_t tv_sec;
+#if defined(GPR_WIN32) && defined(GPR_ARCH_64)
```

Comment:
I don't understand the reason for this: tv_nsec by definition won't get bigger than 1 billion, which will fit into an int on 64bit Windows.

It (might) be better if we use gpr_int32 here instead for some 16 bit platforms, but I don't think that's what this pull request is trying to support.